Session/Conversations/ConversationVC.swift

```diff
@@ -1647,6 +1657,22 @@ final class ConversationVC: BaseVC, LibSessionRespondingViewController, Conversa
             completion?()
         }
     }
+    
+    private func makeInputViewFirstResponder() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) { [weak self] in
+            self?.inputAccessoryView?.becomeFirstResponder()
+        }
+    }
```

Comment:
Since this is only being done once it doesn't need to be a function - we can extract it later if it's reused